assets/js/scripts.js

```diff
@@ -15,4 +15,48 @@ function menuOptions(param) {
   const links = document.getElementById('mobile-menu-container');
   links.style.display = 'none';
 }
-/* eslint-enable */
\ No newline at end of file
+/* eslint-enable */
+
+// client side validation and local storage 
+document.getElementById('form').addEventListener('submit', (e) => {
+  const emailValue = document.getElementById('email').value;
+  const emailError = document.getElementById('error-msg');
+  const regex = /^[a-z]/g;
+  if (regex.test(emailValue)) {
+    emailError.textContent = '';
+    // emptying the local storage
+    localStorage.removeItem('formData');
+  } else {
+    e.preventDefault();
+    emailError.textContent = 'email should be typed in lowercase';
+  }
+});
```

Comment:
- [ ] Great implementation in storing draft of user work but let's make sure we validate that email should be in lowercase if a user enters "tesT@test.com" should be marked as invalid. only pass if it's all in lowercase e.g "test@test.com".